Readme.md

```diff
@@ -78,13 +78,9 @@ You should install *pino-syslog* globally so that it can be used as a utility:
 $ npm install --production -g pino-syslog
 ```
 
-## Command Line Arguments
+## Configuration
 
-There is only one argument available: `--config` (`-c`). This argument is used to load a JSON configuration file.
-
-### Configuration
-
-*pino-syslog* supports configuration via a JSON file. The default options are:
+*pino-syslog* supports configuration via a JSON file and using option flags. The default options are:
```

Comment:
```suggestion
*pino-syslog* supports configuration via a JSON file and/or using option flags; the JSON configuration takes precedence over CLI flags. The default options are:
```

Note: this should probably be reversed. We should probably look for the `--config` flag and load the JSON first and then merge the CLI flags into that configuration.